src/Circlereader/util/caseOptions.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Operator} from '../circle-analysis/circle/operator';
+import {NodeAttributes} from '../type/types';
+
+import {OptionsAttribute} from './optionsAttribute';
```

Comment:
When I reviewed #214, 51 options are found but here are 50 options.. what is missed...?

Reply:
Oh, I forgot about `getConcatEmbeddingAttr`...T.T
I'll add it right away.